src/semantic_version.cr

```diff
@@ -1,14 +1,31 @@
 # Conforms to Semantic Versioning 2.0.0
```

Comment:
Maybe put a newline after this? I think currently it would be shown like this in the docs:

---

#### Conforms to Semantic Versioning 2.0.0 See [https://semver.org/](https://semver.org/) for more information.